### PR TITLE
Removes extra docs components

### PR DESCRIPTION
**Pull request by joeljpoulin**
Removes doctrees from the docs folder and add code in docs script to delete them automatically. 
Adds to documentation the custom demos information. Removes _sources and objects.inv from docs and adds commands in custom docs script.

### CODE REVIEW

**Comment by rdevon** [comment]
I'm all for this. However, shouldn't we also be removing this from history as well? How big are these files and by just having them do we make it harder to clone cortex?

**Comment by joeljpoulin** [comment]
The doctrees folder is 2.4 Mib of size. We can remove them completely from the history, but since it has been added a long time ago, it may be a little hard.

**Comment by rdevon** [comment]
@ReyhaneAskari What's the best way to do this? 2.4Mb isn't too bad I suppose, but it is large relative to the total size of the repo.

**Comment by ReyhaneAskari** [comment]
@rdevon it's really difficult to remove a commit deep in the history. I would say 2.4 is not worth the trouble.

File: docs/html/searchindex.js [hunk 1]
```diff
@@ -1 +1 @@
...
...:2,y_final:2,yaml:[0,7],yet:[],yeung:2,yml:[0,7],you:[6,7],your:6,z_encod:3,z_encoder_arg:3,z_fake:3,z_m:3,z_p:3,z_q:3,z_real:3,zahn:2,zhong:2,zip:2},titles:["cortex package","cortex.built_ins package","cortex.built_ins.datasets package","cortex.built_ins.models package","cortex.built_ins.networks package","cortex.built_ins.transforms package","Develop","Getting Started","Welcome to Cortex2.0","Installation","cortex"],titleterms:{adversarial_autoencod:3,ae_network:4,ali:3,api:[],base_network:4,build:6,built:7,built_in:[1,2,3,4,5],celeba:2,classifi:3,config:[],configur:7,content:[0,1,2,3,4,5],conv_decod:4,convnet:4,cortex2:8,cortex:[0,1,2,3,4,5,10],custom:7,dataset:2,develop:6,discrete_gan:3,document:6,dsprite:2,exampl:7,experi:[],file:7,format:[],from:9,fully_connect:4,gan:3,gener:[],get:7,image_cod:3,imagenet:2,ins:7,instal:9,local:6,main:0,mine:3,model:[3,7],modul:[0,1,2,3,4,5],network:4,nii_dataload:2,option:7,packag:[0,1,2,3,4,5],plugin:0,prerequisit:9,resnet:4,serv:6,server:[],sobel:5,sourc:9,spectralnormlay:4,start:7,submodul:[0,2,3,4,5],subpackag:[0,1],todo:2,torchvision_dataset:2,toyset:2,transform:5,tv_models_wrapp:4,usag:7,util:[2,3,4],vae:3,verifi:[],visdom:9,welcom:8}})
```
Comment (ReyhaneAskari) [inline docs/html/searchindex.js]: Is this file auto-generate too?

File: docs/html/_sources/install.rst.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Installation
```
Comment (ReyhaneAskari) [inline docs/html/_sources/install.rst.txt]: why this file's format is `.rst.txt`?

File: docs/html/_sources/modules.rst.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+cortex
```
Comment (ReyhaneAskari) [inline docs/html/_sources/modules.rst.txt]: same here

File: docs/html/_sources/getting_started.rst.txt [hunk 4]
```diff
@@ -0,0 +1,110 @@
+Getting Started
```
Comment (ReyhaneAskari) [inline docs/html/_sources/getting_started.rst.txt]: `.rst.txt` ?

**Pull request merged**